src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        ColorSupplier colorSupplier;
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+
+        switch (figureNumber) {
+            case 0:
+                colorSupplier = new ColorSupplier();
```

Comment:
should we create new ColorSupplier object all the time?
better create it once in the class FigureSupplier or pass it to the constructor of FigureSupplier from the outside